CleverCloud/clever-components
# Web test runner: Mocha like reporter

## What does this PR do?

- upgrades `@web/test-runner` to `0.18.3` (+ its dependencies to latest versions),
- use `summaryReporter` to get a reporter with the same style as Mocha (only when not in CI),
- filter irrelevant logs.

## How to review?

- Check the commit,
- Check the test results from the latest action, the report should look the same as before without irrelevant browser logs,
- Run tests locally to see the report (should be more detailed than usually).
